buttons_2/style.css

```diff
@@ -0,0 +1,57 @@
+.blue_btn{
+    width: 100px;
+    height: 50px;
+    background-color: rgb(13 69 205);
+    border-radius: 8px;
+    color: white;
+}
+.blue_btn:hover{
+    background-color: red;
+}
+.grey_btn{
+    width: 100px;
+    height: 50px;
+    background-color: rgb(74 80 88);
+    border-radius: 8px;
+    color: white;
+}
+.grey_btn:hover{
+    background-color: orange;
+}
+
+.green_btn{
+    width: 110px;
+    height: 60px;
+    background-color: rgb(22 98 55);
+    border-radius: 8px;
+    color: white;
+}
+.green_btn:focus{
+    background-color: lightgreen;
+    color: black;
+    
+}
+.red_btn{
+    width: 110px;
+    height: 60px;
+    background-color: rgb(171 26 45);
+    border-radius: 8px;
+    color: white;
+}
+.red_btn:focus{
+    background-color: orange;
+    margin-right: 25px;
+    transition: 1s;
+}
+.yellow_btn{
+    width: 60px;
+    height: 60px;
+    border-color: yellow;
+    border-radius: 8px;
+    color: black;
+}
+.yellow_btn:hover{
+    transition: 1s;
+    background-color: yellow;
+    border-radius: 50%;
+}
```

Comment:
Please use DRY (do not repeat yourself) principle.
Create few classes instead of one: one for size, one for colors and one with default params.